osu.Server.Queues.ScoreStatisticsProcessor/Commands/Maintenance/DeleteNonPreservedScoresCommand.cs

```diff
@@ -25,6 +28,7 @@ public async Task<int> OnExecuteAsync(CancellationToken cancellationToken)
             using (var readConnection = DatabaseAccess.GetConnection())
             using (var deleteConnection = DatabaseAccess.GetConnection())
             using (var deleteCommand = deleteConnection.CreateCommand())
+            using (var s3 = S3.GetClient())
```

Comment:
This will throw on missing `S3_KEY` / `S3_SECRET`. Not sure if there should be a flag on the command to skip s3 for testing or something.